crates/asm/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "asm"
```

Comment:
Maybe crate names would be like `strata-asm-common`, `strata-asm-proto-core`, `strata-asm-proto-bridge-v1`, `strata-asm-impl`?